tests/RefreshDatabase.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+trait RefreshDatabase
+{
+    /**
+     * Refresh a conventional test database.
+     */
+    protected function refreshDatabase()
```

Comment:
Yep, we use that command right below this! ⬇️ The difference from what we were doing before is that we only run migrations once at the start of the test suite, and then use transactions to reset it to the initial state between each method.